packager/CMakeLists.txt

```diff
@@ -227,17 +227,21 @@ add_custom_target(packager_test_py_copy ALL
 )
 
 if(NOT SKIP_INTEGRATION_TESTS)
-  add_test (NAME packager_test_py
-    COMMAND ${PYTHON_EXECUTABLE} packager_test.py
-    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-  )
-
-  set_property(TEST packager_test_py
-    PROPERTY ENVIRONMENT "PACKAGER_SRC_DIR=${CMAKE_SOURCE_DIR}"
-  )
-endif()
+  if(NOT Python3_EXECUTABLE)
+    find_package(Python3 COMPONENTS Interpreter REQUIRED)
+  endif()
 
+  if (Python3_EXECUTABLE)
```

Comment:
I would rather see the build fail than skip test coverage without knowing it.  If find_package() fails in CI some day, or even on my workstation, better to know and fix it than to skip these tests, IMO.

Reply:
Makes sense, and I made the find python above required anyway, I've cleaned this up.